### PR TITLE
Merge draft-documents into concepts

### PR DESCRIPTION
**Pull request by adamane**
This will add the swagger documentation to concepts.

### CODE REVIEW

**Comment by adamane** [comment]
## **Please don't delete the branch after merging**

File: documentation/dev/documentation.md [hunk 1]
```diff
@@ -2,4 +2,19 @@
 
 # Documentation
 
+## Swagger Endpoint Documentation
+
+#### YAML
```
Comment (steve-hb) [inline documentation/dev/documentation.md]: Increment just by one

File: documentation/dev/endpoints/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+/.vscode
```
Comment (steve-hb) [inline documentation/dev/endpoints/.gitignore]: move to root directory

File: documentation/dev/endpoints/Concept.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Concept
+
+These Flowcharts can also be viewed by downloading [This](Concept.drawio) file and draging it into the [app.diagrams.net](https://app.diagrams.net/) website.
+
+### Login:
+![Login](endpoints/img/Concept-Login.svg)
```
Comment (steve-hb) [inline documentation/dev/endpoints/Concept.md]: These links are broken

**Review by steve-hb — CHANGES_REQUESTED** [review]

File: documentation/dev/endpoints/Concept.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# Concept
+
+These Flowcharts can also be viewed by downloading [This](Concept.drawio) file and draging it into the [app.diagrams.net](https://app.diagrams.net/) website.
+### Login:
```
Comment (steve-hb) [inline documentation/dev/endpoints/Concept.md]: Increment by one

File: documentation/dev/endpoints/Concept.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# Concept
```
Comment (steve-hb) [inline documentation/dev/endpoints/Concept.md]: file names should be lower case

**Review by steve-hb — CHANGES_REQUESTED** [review]

**Review by steve-hb — APPROVED** [review]

**Pull request merged**